File: views/src/main/java/dlgdev/weighttracker/views/entry/WeightEntryViewHolder.java
package dlgdev.weighttracker.views.entry;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import butterknife.Bind;
import butterknife.ButterKnife;
import dlgdev.weighttracker.R;

public class WeightEntryViewHolder extends RecyclerView.ViewHolder{
	@Bind(R.id.date_view) TextView dateView;
	@Bind(R.id.entry_view) TextView entryView;

	public WeightEntryViewHolder(View itemView) {
		super(itemView);
		ButterKnife.bind(this, itemView);
	}

	public void setEntry(String date, String entry) {
		dateView.setText(date);
		entryView.setText(entry);
	}
}

File: views/src/main/java/dlgdev/weighttracker/views/entry/WeightEntryViewHolder.kt
package dlgdev.weighttracker.views.entry

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.TextView

import butterknife.BindView
import butterknife.ButterKnife
import dlgdev.weighttracker.R

class WeightEntryViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    @BindView(R.id.date_view) lateinit var dateView: TextView
    @BindView(R.id.entry_view) lateinit var entryView: TextView

    init {
        ButterKnife.bind(this, itemView)
    }

    fun setEntry(date: String, entry: String) {
        dateView.text = date
        entryView.text = entry
    }
}
